apps/api/src/app/environments/environments.controller.ts

```diff
@@ -36,6 +36,7 @@ export class EnvironmentsController {
     private getMyEnvironmentsUsecase: GetMyEnvironments
   ) {}
 
+  // TODO: To be removed.
```

Comment:
Fetching all environments for the current user and org should be enough. This will be removed after a week or so to ensure that all web apps have refreshed their source code.